timemachine/cpp/src/context.cu

```diff
@@ -332,12 +332,15 @@ std::array<std::vector<double>, 2> Context::multiple_steps_local(
         gpuErrchk(cudaStreamSynchronize(stream));
 
         // Set the nonbonded potential to compute forces of inner+outer shell.
-        set_nonbonded_potential_idxs(nonbonded_potential, p_num_selected.data[0], d_row_idxs.data, stream);
+        // Commented out as if a particle moves away from the ligand, it may have interactions with
+        // particles that aren't included in frozen
+        // set_nonbonded_potential_idxs(nonbonded_potential, p_num_selected.data[0], d_row_idxs.data, stream);
```

Comment:
curiosity: was this causing issues in practice? Might be worth commenting on whether this is a correctness issue and what a fix could look like

Reply:
I don't have a sense for whether this is happening in practice. Its simply that it could happen and seems safer to avoid the case entirely.

The fix is effectively to move to the Ixn group, as it will be the free vs everything else, so there will be no way for a particle to be non-interacting with another particle in the system.